ssts/segmentation/example.py

```diff
@@ -206,10 +190,49 @@ def main(data_path=None, example_number=None):
         # Plot classification distributions with masks
         mm.show_distributions_together(labels, data)  # all classes plotted together
 
+    # NOTE Wes workflow
+    elif example_number == "7":
+
+        # Apply frequency removal
+        data = mm.apply_frequency_removal(data)
+
+        # Extract outliers
+        outliers = mm.extract_outliers(data)
+        no_outliers_data = np.copy(data)
+        no_outliers_data[outliers==1] = 0
+        mm.show_outliers(data, outliers)
+
+        # Show a-priori distributions
+        mm.show_property_distributions(data, outliers)
+
+        # Run GMM segmentation
+        seg1 = seg_gmm.SegmenterGMM(n_components=2, nonlinear=True)
```

Comment:
@wesleyktatum Nonlinear flag set to True

Reply:
Note that the pca flag can be set to true and dimensionality reduction will be applied to the nonlinear features.